Presentation.Web/Tests/02_Sequential/Local-admin/LocalAdmin.DataProcessing.e2e.spec.ts

```diff
@@ -17,11 +17,11 @@ describe("Local admin is able to toggle DataProcessing", () => {
 
     beforeAll(() => {
         testFixture.enableLongRunningTest();
-        loginHelper.loginAsGlobalAdmin();
-        SystemCatalogHelper.createSystem(systemName);
-        SystemCatalogHelper.createLocalSystem(systemName);
-        loginHelper.logout();
-        loginHelper.loginAsLocalAdmin();
+        loginHelper.loginAsGlobalAdmin()
```

Comment:
hehe and so the story continues :-)

Reply:
mon du kan returne fra den der? hvis den ser at det er et promise så venter den måske med at afslutte beforeAll inden den starter testene

Reply:
Det lader til at ved async metoder bør man fortælle i beforeAll at den er async og så returnere et promise.

Reply:
ja så er det jo bare at returne det  ellers er vi jo lige vidt men fedt at man rent faktisk kan det